app/src/styles/discover.js

```diff
@@ -1,6 +1,15 @@
-import { StyleSheet } from 'react-native';
+import { Dimensions, StyleSheet } from 'react-native';
 import Colors from './colors';
 
+const screenDimension = Dimensions.get('window');
```

Comment:
Might want to place some of this in a utility or somewhere shared; I can imagine much of this being reusable with components as well.